README.md

```diff
@@ -10,27 +10,31 @@ Built with [React](https://facebook.github.io/react/) and [Redux](https://github
 The backend is supported by Docker. If you do not have it already, you will need to install it.
 * For OS X, go [here](https://docs.docker.com/docker-for-mac/).
 * For Windows, go [here](https://docs.docker.com/docker-for-windows/).
-* For Ubuntu and other Linux distributions, go [here](https://docs.docker.com/engine/installation/linux/ubuntulinux/).
+* For Ubuntu and other Linux distributions, install
+[docker](https://docs.docker.com/engine/installation/linux/ubuntulinux/) and
+[docker-compose](https://docs.docker.com/compose/install/).
+
+To [avoid having to use sudo when you use the docker command]
```

Comment:
You are right,, it is not clear that the line starting "To avoid having to use sudo" applies only to the Ubuntu instructions. I will indent that to make it clearer. These are the minimum install instructions for Ubuntu, so I would like to keep them here in the README.md. The lack of these instructions slowed down a large group meeting while people waited for me to puzzle through installation, so I acutely felt their absence.